lazyflow/operators/ioOperators/opNpyFileReader.py

```diff
@@ -25,11 +25,17 @@
 import numpy
 import copy
 
+import logging
+logger = logging.getLogger(__name__)
+
+
 class OpNpyFileReader(Operator):
     name = "OpNpyFileReader"
     category = "Input"
 
     FileName = InputSlot(stype='filestring')
+    InternalPath = InputSlot(stype='string', value="")
```

Comment:
Ah, good point.  Yes, I think it's something we can live with.  To be honest, you've hit upon a more general flaw in the design of lazyflow.  The operator is automatically configured as soon as one of its inputs changes, which isn't always desirable, especially if you intend to change multiple slots in sequence.  In lazyflow's successor, we need to make sure this doesn't happen.

For now, you have to make sure that the operator does something reasonable, or at least gives a clear error message.